api/src/domain_events/executors/broadcast_push_notification.rs

```diff
@@ -29,6 +30,7 @@ impl DomainActionExecutor for BroadcastPushNotificationExecutor {
 impl BroadcastPushNotificationExecutor {
     pub fn new(config: &Config) -> BroadcastPushNotificationExecutor {
         BroadcastPushNotificationExecutor {
+            provider: Some(config.email_templates.custom_broadcast.provider.clone()),
             template_id: Some(config.email_templates.custom_broadcast.template_id.clone()),
```

Comment:
All of these changes can be handled by this change instead:
```suggestion
            template_id: Some(config.email_templates.custom_broadcast.to_string()),
```